frontend/src/scenes/early-access-features/EarlyAccessFeature.tsx

```diff
@@ -387,17 +387,29 @@ interface PersonsTableByFilterProps {
     emptyState?: JSX.Element
 }
 
-function PersonsTableByFilter({ properties, emptyState }: PersonsTableByFilterProps): JSX.Element {
-    const { toggleImplementOptInInstructionsModal } = useActions(earlyAccessFeatureLogic)
-
+export function PersonsTableByFilter({ properties, emptyState }: PersonsTableByFilterProps): JSX.Element {
     const personsLogicProps: PersonsLogicProps = {
         cohort: undefined,
         syncWithUrl: false,
         fixedProperties: properties,
     }
-    const logic = personsLogic(personsLogicProps)
-    const { persons, personsLoading, listFilters } = useValues(logic)
-    const { loadPersons, setListFilters } = useActions(logic)
+
+    return (
+        <BindLogic logic={personsLogic} props={personsLogicProps}>
+            <PersonsTableByFilterScene emptyState={emptyState} />
+        </BindLogic>
+    )
+}
+
+interface PersonsTableByFilterSceneProps {
```

Comment:
hey, thanks Nate, this looks good, except the name `Scene`, which is reserved for standalone components that make up the whole page, I'd rather call this `Component` instead.